integration/spark-common/src/main/scala/org/apache/carbondata/spark/load/DataLoadProcessBuilderOnSpark.scala

```diff
@@ -442,7 +442,8 @@ object DataLoadProcessBuilderOnSpark {
       .map { row =>
         new GenericRow(row.getData.asInstanceOf[Array[Any]])
```

Comment:
Here rdd creation is not required, please remove and please give the reason why we need to use this API instead of existing one in comments.

Reply:
Due to the confliction between data types that is - long in carbonScanRDD and Timestamp in schema, we cannot use the existing API with rdd.